src/containers/edit-profile/profile-tab/ProfileTab.tsx

```diff
@@ -65,16 +69,23 @@ const ProfileTab: FC<ProfileTabProps> = ({ user }) => {
       }
     }
 
-    handleBlocker().catch(console.error)
-  }, [blocker, checkConfirmation, t])
+    handleBlocker().catch((error: string) => {
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.error,
+          message: error
+        })
+      )
+    })
+  }, [blocker, checkConfirmation, t, dispatch])

```

Comment:
Please double check if all dependencies here are needed. For example `t`

Reply:
This useEffect is build in such way that all this dependencies are needed
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/127191734/359dc019-6f11-48e1-bcff-0932749f9b08)